src/Groups/GrpAb.jl

```diff
@@ -326,7 +326,6 @@ function action_homomorphism(Omega::GSetByElements{FinGenAbGroup, S}) where S
 
   # Compute a permutation group `G` isomorphic with `A`.
   phi = isomorphism(PermGroup, A)
-  G = codomain(phi)
 
   # Let `G` act on `Omega` as `A` does.
```

Comment:
Here and a few lines above also `G` should be adjusted as suggested in my other comments

Reply:
```suggestion
  # Let the image of `phi` act on `Omega` as `A` does.
```